eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/RatifySpec.hs

```diff
@@ -52,8 +53,25 @@ relevantDuringBootstrapSpec ::
   forall era.
   ConwayEraImp era =>
   SpecWith (ImpTestState era)
-relevantDuringBootstrapSpec =
+relevantDuringBootstrapSpec = do
   spoVotesForHardForkInitiation
+  initiateHardForkWithLessThanMinimalCommitteeSize
+
+initiateHardForkWithLessThanMinimalCommitteeSize ::
+  forall era.
+  ConwayEraImp era =>
+  SpecWith (ImpTestState era)
+initiateHardForkWithLessThanMinimalCommitteeSize =
+  it "Hard Fork can still be initiated with less than minimal committee size" $ do
+    (hotC :| _) <- registerInitialCommittee
+    passEpoch
+    modifyPParams $ ppCommitteeMinSizeL .~ 2
+    protVer <- getProtVer
+    gai <- submitGovAction $ HardForkInitiation SNothing (majorFollow protVer)
+    submitYesVote_ (CommitteeVoter hotC) gai
+    if bootstrapPhase protVer
+      then isCommitteeAccepted gai `shouldReturn` True
+      else isCommitteeAccepted gai `shouldReturn` False
```

Comment:
In an ideal world, I would prefer to register an initial committee with two members (with minimum committee size set to 2), resign one of them, cast a 'Yes' vote and assert the above. Unfortunately, I don't know how to create a committee with 2 members without using `UpdateCommittee` which can't be used during the bootstrap phase. Any ideas? Or is my current implementation good enough?

Reply:
In an idea world we should start all of our tests with a constitutional committee that has more than one member. So, if you feel adventurous, I'd recommend adding a couple more committee members here: https://github.com/IntersectMBO/cardano-ledger/blob/c40bee5e9c4c7f22822d7e4ec6284d904ec35317/eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs#L263

and go through any test failures and adjust them accordingly. 

That being said, if it proves too difficult (I really don't know how much effort that would be) you can leave this test as-is.

Reply:
Thanks! Exactly what I was looking for, don't know how I missed it. I'll have a go at this in a separate PR then.